src/contributing/debug.md

```diff
@@ -0,0 +1,66 @@
+# Troubleshooting badly behaving apps
+
+Despite the best efforts of Void Linux maintainers, it is possible that you will
+have issues with packages from the official repositories. On such cases, there
+are some steps to follow in order to provide as complete a bug report as
```

Comment:
to follow -> you can follow

We dont require this for a bug report

Reply:
Heh, indeed